dotcom-rendering/src/web/experiments/tests/recipe-structured-data.ts

```diff
@@ -0,0 +1,68 @@
+// This is for an experiment testing SEO performance on recipes before
+// and after structured data is added. Please don't delete - it will be
```

Comment:
Can we add an explicit date from when this file can be deleted. It is astounding how easy it is for code like this to remain for years even when everyone has the best intentions, it happens. I am guilty of this, it is almost impossible to prevent. It is about 500M times easier to add code than it is to delete it.

Reply:
Yeah great shout, my original was too vague. How many 'Will delete later we swear!' comments there must be in the world's code!